docs/guides/preinstall.md

```diff
@@ -0,0 +1,81 @@
+# Pre install steps
+
+These steps are common to most distros. Please refer to both this page and this wiki's install page for your distro.
+
+## Make disk space with Bootcamp Assistant
```

Comment:
Has anyone tried it? Iirc, Bootcamp partitions only when you provide it with the Windows ISO.

Reply:
I had recently reinstalled windows and it asked for the ISO first.

Pretty much similar to this https://youtu.be/Hmm9Q-T0oTo

Reply:
hmm i just tried it and it seems like either they changed it in an update, or i was remembering wrong.

I think that means we can take out this whole section then, which makes things simpler.

Reply:
You may do the editing then.

Reply:
Also review the rest of the PR if you get time.